src/HH4b/processors/corrections.py

```diff
@@ -411,8 +411,14 @@ def get_jec_jets(
         if isData:
             if year == "2022EE" and "Run2022F" in dataset:
                 corr_key = f"{year}NOJER_runF"
+            elif year == "2022EE" and "Run2022E" in dataset:
```

Comment:
Ruff is pointing out this might be an error - did you mean to set the Run2022E corr_key to `...runF"?

Reply:
Ya, this is actually code that was merged already, not sure how it got reverted and entered this PR again.

https://github.com/LPC-HH/HH4b/pull/65#issue-2164062607

But basically, it is a placeholder because the corrections for E don't exist yet (though maybe they do since I made this change a while ago).

Reply:
Will override protections and merge this in since I need to fix the JECs code in any case..